docs/releases/upgrading/upgrade-v11.0.md

```diff
@@ -0,0 +1,85 @@
+# Upgrading from v10 to v11
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v10, please make sure that this is given!
+
+If unsure, run `invenio-cli install` from inside the instance directory before executing the listed steps.
+
+!!! warning "Backup"
+
+    Always backup your database and files before you try to perform an upgrade.
+
+!!! info "Older versions"
+
+    In case you have an InvenioRDM installation older than v10, you can gradually upgrade using the existing instructions to v10 and afterwards continue from here.
+
+## Upgrade Steps
+
+!!! warning "Upgrade your invenio-cli"
+
+    Make sure you have the latest `invenio-cli`, for InvenioRDM v11 the release is v1.0.14.
+
+    ```bash
+    $ invenio-cli --version
+    invenio-cli, version 1.0.14
+    ```
+
+!!! info "Virtual environments"
+    In case you are not inside a virtual environment, make sure that you prefix each `invenio` command with `pipenv run`.
+
+### Python upgrade
+
+As of InvenioRDM v11.0 only Python 3.9 is supported. Therefore you will need to recreate your virtual environment.
+
+the first step is to edit the `<my-site>/Pipfile`:
+
+```diff
+[requires]
+---python_version = "3.9"
```

Comment:
Is the old value also `3.9`?

Reply:
it depends what was chosen in the cookiecutter at the init time. We could have to cover everything:
```suggestion
---python_version = "3.x"
```